.gitignore

```diff
@@ -186,4 +186,4 @@ cover/
 .vscode/
 
 # wandb
-wandb/
+wandb/
```

Comment:
Let's remove all modifications from this file?

Reply:
How do I revert it?
I tried removing the extra line but it's still changed.

Reply:
There's a couple possible ways:
1. I believe you can do this through the github UI by selecting `Discard Changes`:
<img width="1360" alt="Screenshot 2024-10-14 at 3 56 06 PM" src="https://github.com/user-attachments/assets/b584e7bc-cc74-48df-8a08-83be9654920e">
2. Reset the commits and don't add the changes to the file (from git CLI): https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things
3. Copy and paste the file from master,  which hopefully shouldn't have the changes.